templates/includes/sidebar.html

```diff
@@ -1,4 +1,16 @@
 <ul class="sidebar-nav">
+  <li>
+  {% if user.is_authenticated %}
+      User: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a>
```

Comment:
The URL `{% url 'taxi:driver-detail' user.id %}` assumes that the `driver-detail` view uses the user's ID as a parameter. Ensure that the `driver-detail` URL pattern is correctly configured to accept the user ID, or adjust the parameter to match the expected URL pattern.